shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -113,9 +113,8 @@ object JsonLdUtils {
                 expandContext = mapOf(JSONLD_CONTEXT to addCoreContext(contexts))
             }
 
-    fun expandDeserializedPayload(deserializedPayload: Map<String, Any>, contexts: List<String>): Map<String, Any> {
-        return doJsonLdExpansion(deserializedPayload, contexts)
-    }
+    fun expandDeserializedPayload(deserializedPayload: Map<String, Any>, contexts: List<String>): Map<String, Any> =
+        doJsonLdExpansion(deserializedPayload, contexts)
```

Comment:
I know it's more elegant as a one-liner but this should not be part of the PR

Reply:
I do it to correct the error